src/js/common/model/veteran.js

```diff
@@ -1,7 +1,8 @@
 'use strict';  // eslint-disable-line
 // Veteran resource prototype objects. In common so server unittests can access.
 
-const _ = require('lodash');
+const _includes = require('lodash/includes');
```

Comment:
From a C background, leading underscores in identifies makes me cringe...but I assume this is an okay convention in javascript?  If so then great.